src/maison/config.py

```diff
@@ -10,7 +10,7 @@
 
 from maison.errors import NoSchemaError
 from maison.schema import ConfigSchema
-from maison.utils import _collect_configs
+from maison.utils import deep_merge, _collect_configs
```

Comment:
these should be separated out to two lines

```python
from maison.utils import _collect_configs
from maison.utils import deep_merge, 
```

did you enable the pre-commit hooks? `pre-commit install`

Reply:
Done in https://github.com/dbatten5/maison/pull/89/commits/af5bfe6da0a62e0b9749ca2543583f0f3fe5ccf1. Thanks for the tip about `pre-commit install`.